nodejs-functions/src/handlers/automatic-clock-outs/clock_out.js

```diff
@@ -30,7 +28,7 @@ const doClockOut = async (context) => {
   await Promise.all(
     entries.map(async (timeEntryAsJson) => {
       const timeEntry = new TimeEntry(timeEntryAsJson);
-      const { userName, userEmail } = findUserData(users, timeEntry.timeEntry.owner_id);
+      const { userName, userEmail } = findUserData( users, timeEntry.timeEntry.owner_id);
```

Comment:
can you try to remove the blank space? changes should be only what it was actually changed. Maybe it was an auto format?

Reply:
Sure, right away!